external-crates/move/crates/move-compiler/src/parser/mod.rs

```diff
@@ -19,17 +19,32 @@ use anyhow::anyhow;
 use comments::*;
 use move_command_line_common::files::{find_move_filenames, FileHash};
 use move_symbol_pool::Symbol;
-use std::{
-    collections::{BTreeSet, HashMap},
-    fs::File,
-    io::Read,
-};
+use std::collections::{BTreeSet, HashMap};
+use vfs::VfsPath;
+
+/// Contains the same data as IndexedPackagePath but also information about which file system this
+/// path is located in.
```

Comment:
Is this used for anything other than interface files? 
Could you describe the relationship between these paths a bit more?